dist/autosize.js

```diff
@@ -77,6 +77,8 @@
 
 			if (style.boxSizing === 'content-box') {
 				heightOffset = -(parseFloat(style.paddingTop) + parseFloat(style.paddingBottom));
+			} else if (style.boxSizing === 'border-box') {
```

Comment:
This is a fix for:
1. Add several lines to textarea
2. Delete all lines 

Then the text-area will be a few pixels taller than it should be